src/platforms/eglstream-kms/server/egl_output.cpp

```diff
@@ -316,6 +320,42 @@ uint32_t mgek::EGLOutput::crtc_id() const
     return static_cast<uint32_t>(crtc_id);
 }
 
+bool mgek::EGLOutput::queue_atomic_flip(FBHandle const& fb, void const* drm_event_userdata)
```

Comment:
This should probably be shared with `configure`; it's largely the same code. Probably `configure` can be changed to call this.